### PR TITLE
Add sso public key to app cache

### PR DESCRIPTION
**Pull request by TimCsaky**
# Description
- Backend behaves statelessly for JWT verification when realmPublicKey is supplied
- Without realmPublicKey, backend must call OIDC endpoint first

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [ ] I have checked that unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have updated the OpenAPI 3.0 `v*.api-spec.yaml` documentation (if appropriate)
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://ches-pr-68-9f0fbe-dev.pathfinder.gov.bc.ca

File: app/config/custom-environment-variables.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "keycloak": {
     "clientId": "KC_CLIENTID",
     "clientSecret": "KC_CLIENTSECRET",
+    "publicKey": "SERVER_KC_PUBLICKEY",
```
Comment (jujaga) [inline app/config/custom-environment-variables.json:11]: This needs to be `KC_PUBLICKEY` as environment variable

Comment (TimCsaky) [reply app/config/custom-environment-variables.json]: ok. done

**Review by jujaga — APPROVED** [review]

**Review by loneil — APPROVED** [review]
Looking good, lets test out more in DEV to be sure before we promote

**Pull request merged**